src/App.js

```diff
@@ -1,13 +1,109 @@
 import React from 'react';
-
+import uuid from 'uuid';
+import './App.css'
+import Todo from './components/TodoComponents/Todo'
+import TodoForm from './components/TodoComponents/TodoForm'
+import SearchTodo from './components/TodoComponents/searchTodo'
+const todoData = [
+  {
+    task: 'Organize Garage',
+    id: 1528817077286,
+    completed: false
+  },
+  {
+    task: 'Bake Cookies',
+    id: 1528817084358,
+    completed: false
+  }
+];
+const getTodoData= window.localStorage.getItem('todoData');
+console.log(getTodoData)
+let datas = []
+if (getTodoData === null || getTodoData === [] || getTodoData === undefined){
+   window.localStorage.setItem('todoData', JSON.stringify(todoData)) 
+   const tododatas= window.localStorage.getItem('todoData')
+   datas = JSON.parse(tododatas)
+} else {
+  datas= JSON.parse(getTodoData)
+}
 class App extends React.Component {
   // you will need a place to store your state in this component.
   // design `App` to be the parent component of your application.
   // this component is going to take care of state, and any change handlers you need to work with your state
+  constructor(props){
+    super(props)
+    this.state = {
+      todoItem : datas,
+      todo : '',
+      searchTodo : '',
+      displaySearch: []
+    }
+  }
+
+  onChangeHandler = (e)=>{
+    this.setState({todo: e.target.value})
+  }
+
+  addTodo = ()=>{
+    const todoItem = [...this.state.todoItem];
+    if (this.state.todo.trim(' ') !== '') {
+      const todo = {task:this.state.todo, id:uuid(), completed:false};
+      todoItem.push(todo);
+      window.localStorage.setItem('todoData', JSON.stringify(todoItem));
+      this.setState({todoItem:todoItem})
+    }
+  }
+
+  completeTodo = (e)=>{
+    const todos = [...this.state.todoItem]
```

Comment:
It seems this function is taking an id, not an event. Why not just calling the parameter with a meaningful name like id?